ingestion/src/metadata/ingestion/source/database/bigquery/metadata.py

```diff
@@ -55,13 +57,29 @@
 from metadata.ingestion.source.connections import get_connection
 from metadata.ingestion.source.database.column_type_parser import create_sqlalchemy_type
 from metadata.ingestion.source.database.common_db_source import CommonDbSourceService
+from metadata.orm_profiler.orm.registry import CustomTypes
 from metadata.utils import fqn
 from metadata.utils.filters import filter_by_database
 from metadata.utils.logger import ingestion_logger
 
+
+class Bytes(_Binary, TypeEngine):
```

Comment:
_Binary as super class was needed here in order to as bigquery sqlalchemy treats bytes as largebinary and _Binary is part of the largebinary..

Reply:
@ayush-shah what I meant here is in `converter.py` we need to change the `DataType.BINARY` to `CustomTypes.BYTES.value`.

Reply:
that didn't work as expected..

Reply:
I just tested it on my end and it worked

Reply:
so the above change will take care of the data type issue where
Bigquery shows Bytes datatype, but it used to change it to binary on bigquery sqlalchemy during metadata ingestion in OM.
so I overrided it to create a new Data Type just for bigquery which will only change the return type here..

Reply:
with this approach we'll have the same issue with all the DB that are mapping bytes to binary (if that makes sense).

Reply:
got it, I will modify it as per above, but the column's data type will revert to binary ( which is bytes on bigquery ), 

I was thinking, it might affect other sources with datatype as binary, `DataType.BINARY: CustomTypes.BYTES.value` will affect other DBs with binary type, all the binary data types will have to go through the hextobyte conversion..

Reply:
that should be ok. we can tests with other databases that have a binary type